app/src/views/private/components/refresh-sidebar-detail.vue

```diff
@@ -1,14 +1,15 @@
 <script setup lang="ts">
+import { Events, emitter } from '@/events';
+import { computed, onUnmounted, ref, watch } from 'vue';
 import { useI18n } from 'vue-i18n';
-import { computed, ref, watch } from 'vue';
 
 const props = defineProps<{
 	modelValue: number | null;
 }>();
 
 const emit = defineEmits<{
-	(e: 'update:modelValue', value: number | null): void;
-	(e: 'refresh'): void;
+	'update:modelValue': [value: number | null];
```

Comment:
If we wanna make this even nicer, we could use a

```ts
const value = defineModel();
```

and use that instead of the modelValue prop w/ emit 🙂

Reply:
Good idea 🚀